tests/test_standard_flexible_scaler.py

```diff
@@ -6,6 +6,16 @@
 
 
 class ScalerTests(unittest.TestCase):
+    def test_sample_weights(self):
```

Comment:
Could you also add a test with _invalid_ `sample_weights`? What happen if the array shape is wrong? If the type of the parameter is wrong (`sample_weights=False`)?